polars/polars-time/src/date_range.rs

```diff
@@ -110,3 +107,84 @@ pub fn time_range(
     let stop = time_to_time64ns(&stop);
     time_range_impl(name, start, stop, every, closed)
 }
+
+#[cfg(test)]
```

Comment:
Can we move these tests to the `integration` tests? This will save a lot of compile times as every test module is compiled in a separate binary. In the aggregation tests we can accumulate that.